lib/x11utils.pm

```diff
@@ -262,7 +262,7 @@ sub handle_login {
     send_key_until_needlematch('displaymanager', $mykey, 30, 3);
     if (get_var('ROOTONLY')) {
         if (check_screen 'displaymanager-username-notlisted', 10) {
-            record_soft_failure 'bgo#731320/boo#1047262 "not listed" Login screen for root user is not intuitive';
+            record_info 'bgo#731320/boo#1047262 "not listed" Login screen for root user is not intuitive';
```

Comment:
Should we delete these related codes since the bugs are fixed?

Reply:
I suggest to keep the old bug reference and add additional info: "we now use this tag to support login as root user".